symengine/tests/basic/test_sets.cpp

```diff
@@ -464,7 +470,8 @@ TEST_CASE("Complement : Basic", "[basic]")
 
     r1 = set_complement(i1, f2);
     REQUIRE(is_a<Complement>(*r1));
-    REQUIRE(r1->__str__() == "(-oo, oo) \\ {y}");
+    r2 = make_rcp<const Complement>(i1, f2);
```

Comment:
You shouldn't use `make_rcp` directly. For testing you can check that the `universe` of `r1` is equal to `i1` and `container` is equal to `f2`